snapcraft/templates/kernel/ubuntu_kernel_get_pull_commands.sh.j2

```diff
@@ -0,0 +1,31 @@
+# Ubuntu kernel snapcraft pull sources script
```

Comment:
I am not sure to see the value of using jinja templates here. Is that to ease debugging when looking at the rendered scripts?
Otherwise could it be simpler to define env variables for the given inputs (ubuntu_kernel_use_binary_package, release_name, etc.) and keep these scripts "pure shell" scripts? It might make them easier to read and could be automatically checked by shellcheck.